app/forms/concerns/add_anotherable.rb

```diff
@@ -68,7 +69,10 @@ def items_valid?
   def error_message(error, number_of_items, model_class, position)
     key = "activemodel.errors.models.#{model_class.to_s.underscore}.attributes.#{error.attribute}.#{error.type}"
     if number_of_items > 1
-      I18n.t("#{key}_when_many", position:)
+      position_tag = tag.span(position, data: { add_another_role: "errorPosition" })
+      content = I18n.t("#{key}_when_many",
+                       position: position_tag)
```

Comment:
ok - so we're doing this to add `[data-add-another-role="errorPosition"]` & `[data-add-another-role="errorMessage"]`to an error if there is more than one of them, and it used by the **updateErrorMessages** function.